src/mmw/js/src/modeling/tr55/quality/views.js

```diff
@@ -0,0 +1,92 @@
+"use strict";
+
+var Marionette = require('../../../../shim/backbone.marionette'),
+    coreModels = require('../../../core/models.js'),
+    chart = require('../../../core/chart.js'),
+    barChartTmpl = require('../../../core/templates/barChart.html'),
+    resultTmpl = require('./templates/result.html'),
+    tableRowTmpl = require('./templates/tableRow.html'),
+    tableTmpl = require('./templates/table.html');
+
+var ResultView = Marionette.LayoutView.extend({
+    className: 'tab-pane',
+    id: function() {
+        return this.model.get('name');
+    },
+    template: resultTmpl,
+    attributes: {
+        role: 'tabpanel'
+    },
+    regions: {
+        tableRegion: '.quality-table-region',
+        chartRegion: '.quality-chart-region'
+    },
+    modelEvents: {
+        change: 'onShow'
+    },
+    onShow: function() {
+        this.tableRegion.reset();
+        this.chartRegion.reset();
+        if (this.model.get('result')) {
+            var dataCollection = new coreModels.DataCollection(
+                this.model.get('result')
+            );
+
+            this.tableRegion.show(new TableView({
+                collection: dataCollection
+            }));
+
+            this.chartRegion.show(new ChartView({
+                model: this.model,
+                collection: dataCollection
+            }));
+        }
+    }
+});
+
+var TableRowView = Marionette.ItemView.extend({
+    tagName: 'tr',
+    template: tableRowTmpl,
+
+    templateHelpers: function() {
+        return {
+            roundedLoad: this.model.get('load').toFixed(2)
```

Comment:
In @mmcfarland's example, he used 3 decimal points: https://github.com/WikiWatershed/model-my-watershed/pull/412#issuecomment-112840142

I'd say we'd need three, because some values are like 0.0096 something.